src/ZoneServer/Network/Send.Normal.cs

```diff
@@ -358,6 +359,127 @@ public static void UnkDynamicCastEnd(Character character, SkillId skillId, float
 				character.Connection.Send(packet);
 			}
 
+			/// <Summary>
+			/// Creates a skill in client (MONSKL_CRE_PAD)
+			/// </summary>
+			/// <param name="caster"></param>
+			/// <param name="skill"></param>
+			/// <param name="animationName"></param>
+			/// <param name="position"></param>
+			/// <param name="direction"></param>
+			/// <param name="f1"></param>
+			/// <param name="f2"></param>
+			/// <param name="skillHandle"></param>
+			/// <param name="f3"></param>
+			public static void SkillPad(ICombatEntity caster, Skill skill, string animationName,
+				Position position, Direction direction, float f1, float f2, int skillHandle,
+				float f3, bool isVisible = true)
+			{
+				if (ZoneServer.Instance.Data.PacketStringDb.TryFind(animationName, out var packetString))
+					SkillPad(caster, skill, packetString.Id, position, direction, f1, f2, skillHandle, f3, isVisible);
+			}
+
+			/// <summary>
+			/// Creates a pad in client (MONSKL_CRE_PAD)
+			/// </summary>
+			/// <param name="caster"></param>
+			/// <param name="skill"></param>
+			/// <param name="packetStringId"></param>
+			/// <param name="position"></param>
+			/// <param name="direction"></param>
+			/// <param name="f1"></param>
+			/// <param name="f2"></param>
+			/// <param name="skillHandle"></param>
+			/// <param name="f3"></param>
+			public static void SkillPad(ICombatEntity caster, Skill skill, int packetStringId,
```

Comment:
I'm going to be honest with you, I copied this code from Akinue's implementation.  I think you're saying you don't want the overload where you pass in the id instead of the packetstring?  I've removed it.